frontend/src/stores/trainingStore.js

```
@@ -0,0 +1,223 @@
+    pollLogs: async () => {
+        const interval = setInterval(async () => {
+            if (!get().isTraining) {
+                clearInterval(interval)
+                return
+            }
+            try {
+                // Implementation depends on how we expose logs (SSE or polling)
+                // For now, placeholder
+            } catch (err) {
+                console.error(err)
+            }
```

Biome (2.3.13)

[error] This code is unreachable



(lint/correctness/noUnreachable)